stdpopsim/cli.py

```diff
@@ -474,6 +532,36 @@ def add_simulate_species_parser(parser, species):
         choices=[model.id for model in species.demographic_models],
         help=model_help,
     )
+    dfe_help = (
+        "Specify a Distribution of Fitness Effects (DFE) model."
+        "If no demography is specified,"
+        "selection is simulated assuming a constant size demographic model."
+        "Available DFE models:"
+        f"{', '.join(dfe.id for dfe in species.dfes)}"
+        ". Please see --help-dfes for details of these DFE models."
+    )
+    species_parser.add_argument(
+        "--dfe",
+        default=None,
+        metavar="",
+        choices=[dfe.id for dfe in species.dfes],
+        help=dfe_help,
+    )
+    interval_help = (
+        "Specify the interval where selection (given a DFE) is simulated."
+        "Anything outside of the interval is simulated as neutral"
+        "If no interval is specified,"
+        "selection is simulated across the entire region."
+        # say something here about multiple dfes?
+    )
+    species_parser.add_argument(
+        "--interval",
+        default=None,  # the contig length but should we add it here?
+        # type="numpy array", # can I say like that here?
+        metavar="",
+        help=interval_help,
+    )
```

Comment:
It's not clear what the user should pass as the interval. Should this be the fiename of a file containing the intervals?

Reply:
so, for now, we have a string, but later I will work with @andrewkern to get a bed file as an input and the idea is to parse the bed files and introduce selection to any interval from that file.

Reply:
so, the help should explain what/how to put in there (and be accurate for now). Maybe an example, even?

Reply:
For `default`, I'd say to leave it as None but say in the help text what the default is.

Reply:
done